os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -130,26 +130,17 @@ config AMEBASMART_I2CTIMEOTICKS
 endif
 
 config AMEBASMART_I2S
-	bool "Enable Amebasmart I2S"
+	bool "Amebasmart I2S"
 	default n
 
 if AMEBASMART_I2S
-config AMEBASMART_I2S0
-	bool "I2S_0"
+config AMEBASMART_I2S2
```

Comment:
What does the number after AMEBASMART_I2S? 0 and 1 is not nessasary?

Reply:
Please refer his previous reply.
```
 [kaikaikaime]
We have 4 adaptors, I2S0 and I2S1 are internal I2S without GPIO pins which currently will not used based on the current requirement. If there is a need, then we will add. Do you prefer to remove instead?
```

Reply:
Yes, i2s2 is tx and i2s3 is rx respectively.